community_server/src/Model/Table/TransactionsTable.php

```diff
@@ -188,7 +188,7 @@ public function listTransactionsHumanReadable($stateUserTransactions, array $use
                     //echo "prev balance: " . $prev->balance . ", diff_amount: $diff_amount, summe: " . (-intval($prev->balance - $diff_amount)) . "<br>";
                     $final_transactions[] = [ 
                         'type' => 'decay',
-                        'balance' => floatval(-intval($prev->balance - $diff_amount)) / 10000.0,
+                        'balance' => Number::format(floatval(-intval($prev->balance - $diff_amount)) / 10000.0, ['precision' => 2]),
```

Comment:
Can you reuse the conversion - make its own function you call from everywhere to have correct values ?

Reply:
resolved with commit b10886a